packages/samples/vuejswithcli/public/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
I copy pasted all the `App.vue` / `index.html` , `components` etc from the `packages/samples/vuejs` folder.